README.md

```diff
@@ -90,6 +90,30 @@ Here's how to build the StableHLO repo on Linux or macOS:
      -DMLIR_DIR=${PWD}/../llvm-build/lib/cmake/mlir
    ```
 
+   If you are actively developing StableHLO, you may want the following additional
+   CMake settings:
+
+   ```shell
+      cmake .. -GNinja \
+     -DSTABLEHLO_ENABLE_LLD=ON \
```

Comment:
Higher in this file we recommend using LLD for Linux, but not MacOS. Do we want to do the same here?

Reply:
Will fix in subsequent change.